src/Domain/Creators/Cylinder.hpp

```diff
@@ -138,12 +138,14 @@ class Cylinder : public DomainCreator<3> {
         "between LowerBound and UpperBound."};
   };
 
-  struct OuterRadialDistribution {
-    using type = domain::CoordinateMaps::Distribution;
+  struct RadialDistribution {
+    using type = std::vector<domain::CoordinateMaps::Distribution>;
     static constexpr Options::String help = {
-        "Select the radial distribution of grid points in the outermost "
-        "spherical shell. The 'RadialPartitioning' must have at least one "
-        "entry if you select a distribution other than 'Linear'."};
+        "Select the radial distribution of grid points in each cylindrical "
+        "shell. The innermost shell must have a 'Linear' distribution because "
+        "it changes the circularity. The 'RadialPartitioning' determines the "
```

Comment:
Changes 'in' circularity, maybe? Since the cirularity is a property of the shell/mesh.

Reply:
Changed in the last fixup commit